edu.wpi.first.wpilib.plugins.cpp/resources/templates/command-based/Robot.cpp

```diff
@@ -51,9 +52,9 @@ class Robot: public frc::IterativeRobot {
 			autonomousCommand.reset(new ExampleCommand());
 		} */
 
-		autonomousCommand.reset(chooser.GetSelected());
+		autonomousCommand = chooser.GetSelected();
 
-		if (autonomousCommand.get() != nullptr) {
+		if (autonomousCommand) {
```

Comment:
Please check against nullptr. It's a pointer, not a bool, and that's what PacGoat does.